doc/input.md

```diff
@@ -161,15 +161,24 @@ Warning: `HasTapableComponents` uses an advanced gesture detector under the hood
 
 Just like with `Tapable`, Flame offers a mixin for `Draggable`.
 
-By adding the `HasDraggableComponents` mixin to your game, and by using the mixin `Draggable` on your components can override the following methods, enabling an easy to use drag event on your components.
-
-Note that the `DragEvent.initialPosition` doesn't take any padding or margin outside the game widget into account, so if you are not running the game in full screen you'll have to take that into account.
+By adding the `HasDraggableComponents` mixin to your game, and by using the mixin `Draggable` on your components, they can override the simple methods that enable an easy to use drag api on your components.
 
 ```dart
-  void onReceiveDrag(DragEvent details) {}
+  void onDragStarted(int pointerId, Vector2 startPosition) {}
+  void onDragUpdated(int pointerId, DragUpdateDetails details) {}
+  void onDragEnded(int pointerId, DragEndDetails details) {}
+  void onDragCanceled(int pointerId) {}
 ```
 
-Minimal component example:
+Note that all events that a uniquely generated pointer id so you can, if desired, distinguish between different simultaneous drags.
```

Comment:
"that a uniquely generated pointer id" -> "receives a uniquely generated pointer id"